test/sagemaker_tests/pytorch/training/integration/sagemaker/test_smdataparallel.py

```diff
@@ -127,6 +129,8 @@ def test_smdataparallel_mnist(ecr_image, sagemaker_regions, efa_instance_type, t
     """
     Tests smddprun command via Estimator API distribution parameter
     """
+    if ecr_image.split("/")[1].startswith("pr-pytorch-trcomp-training"):
```

Comment:
the ecr repo name changes in the rc build. use `endswith("pytorch-trcomp-training")`

Reply:
Also add this logic in `validate_or_skip_smdataparallel` function